frontend/src/Components/CommentComponent.module.scss

```diff
@@ -1,4 +1,5 @@
 .comment {
+  overflow-anchor: none;
```

Comment:
I don't think this is a good idea to make this a default. Instead, I'd suggest creating a separate style `.no-anchor` and adding and removing it as needed before and after scroll. The style could be applied even to some common ancestor of `.comment` and `.post`.